draft-ietf-quic-applicability.md

```diff
@@ -451,16 +451,21 @@ firewalls that rely on the port number for application identification.
 
 # Connection Migration
 
-QUIC supports connection migration. Even if lower-layer addresses (usually the
-4-tuple of IP addresses and ports) changes, QUIC packets can still be associated
-with an existing connection based on the Connection ID (see also section
-{{connid}}) in the QUIC header, if present. This supports cases where address
-information changes due to e.g. NAT rebinding or change of the local interface.
+QUIC supports connection migration by the client. If a lower-layer address
+changes, a QUIC endpoint can still associate packets with an existing connection
+based on the Connection ID (see also section {{connid}}) in the QUIC header,
+if present. This supports cases where address information changes due to e.g.
```

Comment:
Avoid "e.g." used in this way.  Try to say things like "such as" instead.

Reply:
```suggestion
if present. This supports cases where address information changes, such as
```